src/ralph/data_importer/management/commands/importer.py

```diff
@@ -79,8 +83,49 @@ def add_arguments(self, parser):
             default='utf-8',
             help="Output encoding",
         )
+        parser.add_argument(
+            '--dir',
+            help="Csv files directory",
+        )
+        parser.add_argument(
+            '--zipfile',
+            help="ZIP File name",
+        )
+
+    def from_zip(self, options):
+        with open(options.get('zipfile'), 'rb') as f:
+            out_path = tempfile.mkdtemp()
+            z = zipfile.ZipFile(f)
+            for name in z.namelist():
+                z.extract(name, out_path)
+            options['dir'] = out_path
+            options['skipid'] = True
+            self.from_dir(options)
+
+    def from_dir(self, options):
+        file_list = []
+        for path in glob.glob(os.path.join(options.get('dir'), '*.csv')):
+            base_name = os.path.basename(path)
+            file_name = os.path.splitext(base_name)[0].split('_')
+            file_list.append({
+                'model': file_name[1],
+                'path': path,
+                'sort': file_name[0]
+            })
+        file_list = sorted(file_list, key=lambda x: x['sort'])
```

Comment:
(related with https://github.com/allegro/ralph/pull/1456, but it's already merged and i think about post factum)

https://github.com/allegro/ralph/pull/1456/files#diff-78029b48156ee44655b1cf8e84dce40fR127 - add filling file number with zeros - otherwise this sorting will not work. Example:

```
>>> sorted(['1_sth.csv', '2_sth.csv', '10_sth.csv'])
['10_sth.csv', '1_sth.csv', '2_sth.csv']
```